### ISSUE TITLE
nested `:fork`

### ISSUE THREAD

**Issue by GamingKing2436** [radian-software/straight.el#1117]
i was recently attempting to convert my emacs to `straight.el` and i liked the `:fork` feature. naturally i attempted to nest them as such:
```el

(use-package elastic-table
  :straight
  (
   :host github
   :repo "tenbillionwords/spaceship-mode"
   :fork
   (
    :repo "jyp/elastic-modes"
    :fork
    (
     :repo "GamingKing2436/elastic-modes"))))
;; elastic tables yay
```
but it used the [`jyp/elastic-modes`](https://github.com/jyp/elastic-modes) repo instead. looking closer at the docs apperently this isn't a bug.

it would be really cool if they could be nested. 😀

**Comment by raxod502** [comment]
Yes, that is reasonable, it is not currently supported but could be with sufficient work.
